ihatemoney/models.py

```diff
@@ -476,7 +477,8 @@ def verify_token(token, token_type="auth", project_id=None, max_age=3600):
 
         :param token: Serialized TimedJsonWebToken
         :param token_type: Either "auth" for authentication (invalidated when project code changed),
-                        or "reset" for password reset (invalidated after expiration)
+                        or "reset" for password reset (invalidated after expiration),
+                        or "feed" for project feeds (invalidated when project code changed)
```

Comment:
Add a test that checks that this token is indeed invalidated.

Reply:
Done.